pyro/distributions/testing/rejection_gamma.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, alpha):
         if alpha.data.min() < 1:
             raise NotImplementedError('alpha < 1 is not supported')
         self.alpha = alpha
-        self._standard_gamma = Gamma(alpha, alpha.new([1]).expand_as(alpha))
+        self._standard_gamma = Gamma(alpha, torch.empty_like(alpha).fill_(1).expand_as(alpha))
```

Comment:
@fritzo - Had to make a small change to support scalars. Is there a more concise way to do this?

Reply:
How about `alpha.new([1]).squeeze().expand_as(alpha)`?